### PR TITLE
Try ccache install via action

### LINKED ISSUE #2497: Windows ccache
https://github.com/scipp/scipp/pull/2320 made changes to our conda env files and the github workflow to use ccache on Windows. https://github.com/hendrikmuhs/ccache-action just released 1.2 which adds experimental Windows support.

Check if the actions installs ccache on its own. If so, revert part of the changes done to do it via conda.

### PR DESCRIPTION
**Pull request by SimonHeybrock**
Closes #2497

### CODE REVIEW

File: .github/workflows/pr_and_main.yml [hunk 1]
```diff
@@ -79,13 +79,13 @@ jobs:
       - run: echo "$CONDA/envs/buildenv/Library/bin" >> $GITHUB_PATH
         if: ${{ contains(matrix.os, 'windows-latest') }}
 
-      - run: ccache --version
-
       - name: ccache
-        uses: hendrikmuhs/ccache-action@fedaed4959d48938cb5d4be1017da9649a151482
+        uses: hendrikmuhs/ccache-action@v1.2
```
Comment (SimonHeybrock) [inline .github/workflows/pr_and_main.yml:83]:
```suggestion
        uses: hendrikmuhs/ccache-action@v1.2.1
```

File: lib/CMakeLists.txt [hunk 2]
```diff
@@ -74,6 +74,8 @@ find_program(CCACHE ccache)
 if(CCACHE)
   set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
 endif()
+message(STATUS "Tried to find ccache, found:")
+message(STATUS ${CCACHE})
```
Comment (jl-wynen) [inline lib/CMakeLists.txt:78]: What about these? Are they left over from debugging?

Comment (SimonHeybrock) [reply lib/CMakeLists.txt:78]: Yes, but I decided to keep them anyway, unless you mind?

Comment (jl-wynen) [reply lib/CMakeLists.txt:78]: No, just curious.

**Pull request merged**